mingw/ci_make_apps.sh

```diff
@@ -14,7 +14,7 @@ export CXX="x86_64-w64-mingw32-g++"
 export CC="x86_64-w64-mingw32-gcc"
 # condor
 export MINGW_FLAGS="-Dflockfile=_lock_file -Dfunlockfile=_unlock_file"
-export MINGW_CURL="-lcurl -lwinmm -lws2_32 -lpthread -lssl -lcrypto -lcrypt32 -lws2_32 -lzlib -ladvapi32 -lcrypt32"
+export CURL_EXTRA_LDFLAGS="-lcurl -lwinmm -lws2_32 -lpthread -lssl -lcrypto -lcrypt32 -lws2_32 -lzlib -ladvapi32 -lcrypt32"
```

Comment:
'-lws2_32' and '-lcrypt32' are specified twice. Please remove duplicates.

Reply:
I can try to remove them, but:
- This is the exact flags taken from makefile. 
- Sometimes duplicate flags is needed to specify the compiler to use this flag for each compiler, especially in old compilers. 

My recommendation is to leave it as it is.